src/streams/CreateStreamScreen.js

```diff
@@ -22,8 +22,8 @@ export default function CreateStreamScreen(props: Props): Node {
   const ownEmail = useSelector(getOwnEmail);
 
   const handleComplete = useCallback(
-    (name: string, description: string, isPrivate: boolean) => {
-      api.createStream(auth, name, description, [ownEmail], isPrivate);
+    async (name: string, description: string, isPrivate: boolean) => {
+      await api.createStream(auth, name, description, [ownEmail], isPrivate);
```

Comment:
Should we also preemptively add a try/catch and show a toast here?

Reply:
Yeah, good thought -- let's handle this the same way.